Makefile

```diff
@@ -78,13 +78,13 @@ verify:
 	@./scripts/update-codegen.sh && ./scripts/add-openapi-bits.sh && ./scripts/verify-codegen.sh
 
 # Individual targets are found in .nsm.mk
-docker-build: docker-build-netmesh-test docker-build-netmesh docker-build-nsm-init docker-build-nse docker-build-test-dataplane docker-build-sidecar-injector
+docker-build: ;
 
 # Individual targets are found in .nsm.mk
-docker-push: docker-login docker-push-netmesh docker-push-test-dataplane docker-push-nsm-init docker-push-nse docker-push-sidecar-injector
+docker-push: docker-login
 
 # Individual targets are found in .nsm.mk
-docker-save: docker-save-netmesh docker-save-test-dataplane docker-save-nsm-init docker-save-nse docker-save-sidecar-injector docker-push-nsmd-k8s
+docker-save: docker-push-nsmd-k8s
```

Comment:
I believe docker-nsmd-k8s-push should be handled in the build-nsmd-k8s circleci job.